tests/grid/cell_id_08.cc

```diff
@@ -105,8 +105,9 @@ benchmark(const unsigned int n_global_refinements)
     AssertThrow(cell_ids == deserialized,
                 ExcMessage("Serialization with boost binary archives failed."));
     deallog << "Serialization with boost binary archives successful."
-            << std::endl
-            << "  Buffer size in bytes: " << size_in_bytes(buffer) << std::endl;
+            << std::endl;
+    std::cout << "  Buffer size in bytes: " << size_in_bytes(buffer)
+              << std::endl;
```

Comment:
Did you mean to remove these two lines? The output you produce here is identical to the code before -- you just split it over two statements.

Reply:
The output is not identical, is it? The size is now written to `std::cout` as suggested in https://github.com/dealii/dealii/pull/11425#issuecomment-751531382.

Reply:
Oh, I see. I missed that part. I think it's pointless to write to `std::cout` -- nobody is ever going to see that output -- but let's just go with it for now. I'm going to remove this again in a follow-up. For now, let's just make the test failure go away.